gaurav/src/routes/Root.tsx

```diff
@@ -1,19 +1,40 @@
-import { Link } from "react-router-dom";
+import { Calendar, Clock, House, } from "lucide-react";
+import { Link, Outlet } from "react-router-dom";
 export const Root = () => {
   return (
     <>
-      <div className="flex justify-center items-center bg-purple-500">
-        <ul className="flex justify-center items-center gap-3 p-3">
-          <li className="text-white font-medium">
-            <Link to="/">Home</Link>
-          </li>
-          <li className="text-white font-medium">
-            <Link to="/login">Login</Link>
-          </li>
+      <div className="flex justify-center items-center fixed bottom-0 w-full">
+        <ul className="flex justify-between md:justify-center items-center bg-primary gap-6 p-2 px-4 w-full">
```

Comment:
Menus can be improved.... it should have more padding and better UI. Also add some content on home route